snapshot/restore/local_fs_output.go

```diff
@@ -281,6 +286,28 @@ func (o *FilesystemOutput) shouldUpdateTimes(local, remote fs.Entry) bool {
 	return !local.ModTime().Equal(remote.ModTime())
 }
 
+func (o *FilesystemOutput) getStreamCopier(ctx context.Context) streamCopier {
+	if o.Sparse {
+		if !isWindows() {
+			return func(w io.WriteSeeker, r io.Reader) (int64, error) {
+				s, err := stat.GetBlockSize(o.TargetPath)
```

Comment:
nit: we can do this once per FilesystemOutput since it's not going to change. Perhaps call it right at the very beginning?

Reply:
Done: moved `getStreamCopier()` to be public and called only once when first constructing `FilesystemOutput`.